### ISSUE TITLE
Allow extended prior sequences (e.g. full-length 16S)

### ISSUE THREAD

**Issue by alexthibodeau** [benjjneb/dada2#496]
I am extremely interested in using priors.

What is the format of this file?

fasta, amplicon style (for example, V4 region only) 
or fasta full 16S read?

**Comment by benjjneb** [comment]
`priors` are provided as a character vector of sequences. They can come from a file, but should be read into a character vector before being passed to the `dada` function.

Right now, the priors have to be "amplicon style", i.e. the same sequence fragment that would be found in the reads. (In the future, I'd like to relax that requirement to allow exact matching to longer priors like full-length 16S sequences).

**Comment by marcomeola** [comment]
Any idea in which update of dada2 the exact matching to longer priors like full-length 16S sequences could be available?

**Comment by benjjneb** [comment]
> Any idea in which update of dada2 the exact matching to longer priors like full-length 16S sequences could be available?

No, but step one would be to have an open enhancement request on this issue. I'll convert this thread to that.
